### DISCUSSION TITLE
Microsoft Copilot

### DISCUSSION THREAD

**Discussion by jaygz316** [selfhst/icons#132] [New Icon Request]
[copilot icon](https://www.google.com/url?sa=i&url=https%3A%2F%2Fen.m.wikipedia.org%2Fwiki%2FFile%3AMicrosoft_365_Copilot_Icon.svg&psig=AOvVaw27m94WyPkNMAabzks-P_Wh&ust=1731871790169000&source=images&cd=vfe&opi=89978449&ved=0CBEQjRxqFwoTCNjAzarL4YkDFQAAAAAdAAAAABAE)

**Comment by shollyethan** [comment]
Microsoft Copilot has been added - thanks!
